pegasus/sites.v3/code.org/views/dance_teacher_resources.haml

```diff
@@ -30,5 +41,5 @@
           .resource-description
             = resource[:description]
           %a{href: resource[:link]}
-            %button.gray-button
+            %button.gray-button{:id => "#{resource[:id]}"}
```

Comment:
ids are unnecessary here and in the resources hash, but I guess there's no harm in having them.  Do you have an anticipated reason you'll use them later?

Reply:
The ids are not necessary.  There were added as part of an initial solution to the problem.  Thanks!